tests/jsTests/validationSpec.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Suite : validateNumber
+ * Description : Test suite to validate comrades'phone number
+ */
+describe('validateNumber', function() {
+    it('Should be phone number', function() {
+        expect(validation.isPhoneNumber('342345235')).toBeTruthy();
+    });
+
+    it('Should not be phone number', function() {
+        expect(validation.isPhoneNumber('324fdgfd215235')).not.toBeTruthy();
+    });
+});
+
+/**
+ * Suite : validateEmail
+ * Description : Test suite to validate users' email address
+ */
+describe('validateEmail', function() {
+    it('Should be email', function() {
+        expect(validation.isEmailAddress('abc@def.com')).toBeTruthy();
+    });
+
+	it('Should be email', function() {
+        expect(validation.isEmailAddress('mail.google.com')).not.toBeTruthy();

```

Comment:
Write some more negative tests. 
example : email@111.222.333.44444 this one is invalid
email@123.123.123.123 this one is valid 
email@-domain.com not valid
think of some edge cases if any